base/applications/network/net/cmdUser.c

```diff
@@ -1255,7 +1255,7 @@ cmdUser(
             goto done;
         }
     }
-    else if (bAdd && !bDelete)
```

Comment:
What improvements does such change bring? Insofar you haven't described anything about these kind of changes.

Reply:
We may not check both of these flags because we are handle a situation where badd and bDelete are true.
So if bAdd is true, bDelete is always false.
```c++
if (bAdd && bDelete)
    {
        result = 1;
        goto done;
    }
```